packages/terra-form/src/components/Textarea.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Textarea.scss';
+
+const propTypes = {
+  /**
+   * How many columns the textarea should have
+   */
+  cols: PropTypes.number,
```

Comment:
Do you think the names cols and rows will confuse the consumer to think they should be inputing arrays instead of number values?